konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/pipeline/graph/GraphStep.java

```diff
@@ -75,12 +77,31 @@
      */
     boolean hasStep();
 
+    @Schema(description = "Determines if the graph step has a pipeline step associated with it.")
+    default boolean getHasStep() {
+        return hasStep();
+    }
```

Comment:
This shouldn't be in the API - it's not configurable. It depends only on the type of GraphStep

Reply:
I've made this field readonly now